Tasks/NugetPackager/task.json

```diff
@@ -46,6 +43,61 @@
             "required":false
         },
         {
+            "name": "versionByBuild",
+            "type": "pickList",
+            "label": "Automatic package versioning",
+            "defaultValue":"false",
+            "helpMarkDown": "If you choose 'Use semantic versioning', this will generate a [SemVer](http://semver.org/spec/v1.0.0.html)-compliant version formatted as `X.Y.Z-ci-datetime` where you choose X, Y, and Z.\n\nIf you choose 'Use an environment variable', you must select an environment variable and ensure it contains the version number you want to use.\n\nIf you choose 'Use the build number', this will use the build number to version you package. **Note:** Under General set the build format to be '$(BuildDefinitionName)_$(Year:yyyy).$(Month).$(DayOfMonth)$(Rev:.r)'.",
+            "required":true,
+            "groupName": "versioning",
+            "options": {
+                "false": "Off",
+                "byPrereleaseNumber": "Use semantic versioning",
```

Comment:
Open to suggestions. Before I had said "generate a prerelease version string" which was pretty terrible but arguably more correct.